guides/quickstart.md

```diff
@@ -4,426 +4,278 @@ title: Quickstart
+  test('it just works', async function(assert) {
+    await render(hbs`{{quickstart-calculator}}`);
 
-  assert.equal(page.error, '');
+    await c.clickDigit(2);
+    await c.plus();
+    await c.clickDigit(3);
+    await c.equals();
 
-  Ember.run(() => {
-    this.set('error', 'Invalid credentials');
+    assert.equal(c.value, 5);
   });
 
-  assert.equal(page.error, 'Invalid credentials');
-});
-```
-
-Let's take a look at the changes:
+  test('it works with chaining', async function(assert) {
+    await render(hbs`{{quickstart-calculator}}`);
 
-- In the test's `beforeEach()` hook we set the page's test context with `page.setContext(this)`. That tells the page object to use the test's `this.$()` to find elements, instead of Ember's global acceptance test helpers.
-- In the `afterEach()` hook, we call `page.removeContext()` to clear the test context from the page object.
-- The rest of the changes are the same as in our acceptance tests: After you set the test's `this` context on the page object, you can use the page object as before. (The one exception is `page.visit()`, which doesn't work in component tests since we don't have access to a router.)
+    await c
+      .clickDigit(2)
+      .plus()
+      .clickDigit(3)
+      .equals();
 
-As in our acceptance tests, we can DRY things up a bit more by grouping actions together into methods that describe specific user flows. For example, in the first test we can use our `page.loginSuccessfully()` method to eliminate a few lines of code:
+    assert.equal(c.value, 5);
+  });
+```
 
-```js
-test('calls submit action with correct username and password', function(assert) {
-  assert.expect(2);
+Our new page object encapsulates all the possible interactions we'll need when testing our calculator with an idiomatic API that abstracts away the necessary DOM traversal and interaction. No more littering our test with brittle CSS selectors that hamper readability!
 
-  function submit(username, password) {
-    assert.equal(username, 'admin');
-    assert.equal(password, 'secret');
-  }
+## Pages
 
-  this.set('submit', submit);
+For comprehensive testing of our application, we must rely on more than testing individual components in isolation (ie acceptance testing). We can map each "page" in our app (route + template) to a `page-object`, composing various `page-object-component`s together to form a complete representation of page.
 
-  this.render(hbs`{{login-form
-    submit=(action submit)
-  }}`);
+Let's take a look at how we might generate a page-object.
 
-  page.loginSuccessfully();
-});
 ```
+$ ember generate page-object my-page
 
-And that's it! Our integration and acceptance tests are cleaner, more maintainable and easier to read.
+installing
+  create tests/pages/my-page.js
+```
 
-## Appendix
+The generator created a file inside the directory `/tests/pages`. It allows to easier distinguish pages between components, which are located under `/tests/pages/components/`.
```

Comment:
"Using this directory allows us to more easily distinguish pages from components, which are located under `/tests/pages/components/`"